### DISCUSSION TITLE
Add an option for control which Pit value is displayed in the RangeSlider

### DISCUSSION THREAD

**Discussion by samouss** [algolia/instantsearch#5209] [Ideas]
**Do you want to request a *feature* or report a *bug*?**

Feature.

**Feature: What is your use case for such a feature?**

Right now we display the values at 0, 50 and 100 percent. In some cases we might want to display more / less values in the pits. We should be able to control where & when the pits are append.

**Without control (current)**

![screen shot 2018-03-19 at 16 30 13](https://user-images.githubusercontent.com/6513513/37605130-dcd09dda-2b92-11e8-8c2b-9f76cb94c208.png)

**With control (wanted)**

![screen shot 2018-03-19 at 16 30 22](https://user-images.githubusercontent.com/6513513/37605139-dfc9eac8-2b92-11e8-85a8-46661eff04f8.png)

**What is the version you are using? Always use the latest one before opening a bug issue.**

Right now the generation is computed from an array `[0, 50, 100]`. Maybe we should expose this an option or a function that return an array like this one.

-----

Linked to https://github.com/algolia/instantsearch.js/issues/2755

**Comment by Haroenv** [comment]
related: https://github.com/algolia/instantsearch.js/pull/2277

**Comment by bobylito** [comment]
@samouss can you check the PR from @Haroenv and especially give your take on this [comment](https://github.com/algolia/instantsearch.js/pull/2277#issuecomment-336159489).
